Makefile

```diff
@@ -29,23 +19,14 @@ deploy: manifests
 manifests:
 	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
 	sed -i'' -e 's@^- manager_auth_proxy_patch.yaml.*@#&@' config/default/kustomization.yaml
+	sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./config/default/manager_image_patch.yaml
 	kustomize build config/default/ > provider-components.yaml
 	echo "---" >> provider-components.yaml
 	kustomize build vendor/sigs.k8s.io/cluster-api/config/default/ >> provider-components.yaml
 
-# Generate code
-generate:
```

Comment:
generate will be called by either docker-build or test steps inside the dockerfile, so this should no longer be needed.

Reply:
I think we need this to get the local files so that we can build locally with `go build`.